libs/application-generic/src/usecases/execute-bridge-request/execute-bridge-request.usecase.ts

```diff
@@ -256,8 +214,19 @@ export class ExecuteBridgeRequest {
     switch (workflowOrigin) {
       case WorkflowOriginEnum.NOVU_CLOUD:
         return `${this.getApiUrl()}/v1/environments/${environmentId}/bridge`;
-      case WorkflowOriginEnum.EXTERNAL:
+      case WorkflowOriginEnum.EXTERNAL: {
+        if (!environmentBridgeUrl) {
+          throw new BadRequestException({
+            code: BRIDGE_EXECUTION_ERROR.INVALID_BRIDGE_URL.code,
+            message:
+              BRIDGE_EXECUTION_ERROR.INVALID_BRIDGE_URL.message(
+                environmentBridgeUrl,
+              ),
+          });
+        }
```

Comment:
Throw an exception here explicitly if trying to request to an external Bridge URL when it doesn't exist on the environment.